packages/fast-components-react-base/src/image/examples.data.tsx

```diff
@@ -1,38 +1,41 @@
 import * as React from "react";
-import Image, { IImageHandledProps, IImageManagedClasses, IImageUnhandledProps } from "./image";
+import Image, { IImageHandledProps, IImageManagedClasses, IImageUnhandledProps, ImageSlot } from "./image";
 import schema from "./image.schema.json";
 import Documentation from "./.tmp/documentation";
 import { IComponentFactoryExample } from "@microsoft/fast-development-site-react";
 
+const classes: IImageManagedClasses = {
+    managedClasses: {
+        image: "image",
+        image__picture: "picture",
+        image_img: "img"
+    }
+};
+
+const sourceElements: JSX.Element[] = [
+    (<source srcSet="https://placehold.it/350x350/2F2F2F/171717" media="(min-width: 1400px)" slot={ImageSlot.source}/>),
+    (<source srcSet="https://placehold.it/300x300/2F2F2F/171717" media="(min-width: 1084px)" slot={ImageSlot.source}/>),
+    (<source srcSet="https://placehold.it/200x200/2F2F2F/171717" media="(min-width: 768px)" slot={ImageSlot.source}/>),
+    (<source srcSet="https://placehold.it/100x100/2F2F2F/171717" media="(min-width: 540px)" slot={ImageSlot.source}/>),
+    (<source srcSet="https://placehold.it/75x75/2F2F2F/171717" media="(min-width: 0px)"slot={ImageSlot.source}/>)
+];
```

Comment:
Looks like you had a bad merge. These shouldn't be duplicated as part of this commit.